testsuite/path.c

```diff
@@ -184,11 +184,15 @@ WRAP_1ARG(DIR*, NULL, opendir);
 WRAP_1ARG(int, -1, chdir);
 
 WRAP_2ARGS(FILE*, NULL, fopen, const char*);
-WRAP_2ARGS(FILE*, NULL, fopen64, const char*);
 WRAP_2ARGS(int, -1, mkdir, mode_t);
 WRAP_2ARGS(int, -1, access, int);
 WRAP_2ARGS(int, -1, stat, struct stat*);
 WRAP_2ARGS(int, -1, lstat, struct stat*);
+
+WRAP_OPEN();
+
+#if HAVE_DECL___GLIBC__
```

Comment:
not sure we can rely on __GLIBC__ and it'd be better to check case by case. Can't we simply

```
#ifndef fopen64
WRAP_2ARGS(FILE*, NULL, fopen64, const char*);
#endif
```

and so on? Because if it's coming from the header as a define, then we shouldn't wrap it (as we obviously can't).